keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -119,19 +118,6 @@ class ApplicationService(
       targetEnvironment = targetEnvironment)
   }
 
-  /**
-   * Returns a list of [ResourceSummary] for the specified application.
-   *
-   * This function assumes there's a single delivery config associated with the application.
-   */
-  fun getResourceSummariesFor(application: String): List<ResourceSummary> =
```

Comment:
Also moved to `ResourceService`.

Reply:
Why? This is specifically in support of the application summary endpoint, I think it makes sense to stay here with all of the related functions

Reply:
Yeah, I didn't like this either, but the reason I moved it was that the `Resource<*>.toResourceSummary()` extension function has to call `getStatus(id)` for the resource, which, in turn, depends on other beans. Note that this function here was just delegating to `ResourceRepository` previously, where all those other functions used to live.

The alternative was to substitute `resourceService` for `repository` here, but that will introduce a dependency between `ApplicationService` and `ResourceService` which also felt a bit awkward, so I opted for moving it and just injecting both beans into `ApplicationController`.

Happy to undo this if you think the dependency between the two service beans is OK. Or let me know if you have any other ideas!

Reply:
Hmm I see. Your original point about the `ApplicationService` was that the controller should be super light, and all the logic should be in the service. I agree with that, and I think based on that reasoning it would make sense to wire in the new history/event service into this class (because you're using that information to generate the application views).